Assets/HoloToolkit/Input/Scripts/Focus/FocusManager.cs

```diff
@@ -128,7 +128,7 @@ private void OnValidate()
 
         #region Data
 
-        private class PointerData
+        private class PointerData : PointerResult
```

Comment:
I'm still trying to figure out if we should just make `PointerData` public and serialize it, without having to make the `PointerResult` class.

Thoughts?

Reply:
When I initially created PointerResult it was the path of least resistance and I didn't like it. But it's growing on me. Pointers can see the result of their raycasts without having to sift through (or corrupt) the private members that only FocusManager cares about, and we don't have to update multiple objects.

Later on I wouldn't mind refactoring it away but I think we should keep it for now, if only because it's relatively undisruptive.